sal_interfaces/CCCamera/CCCamera_Events.xml

```diff
@@ -1,1128 +1,1407 @@
+        </item>
+        <item>
+            <EFDB_Name>raft_TempControl_awGain</EFDB_Name>
+            <Description>anti-windup gain</Description>
+            <IDL_Type>double</IDL_Type>
+            <Units>unitless</Units>
+            <Count>1</Count>
+        </item>
+        <item>
+            <EFDB_Name>raft_TempControl_basePower</EFDB_Name>
+            <Description>base power input</Description>
+            <IDL_Type>double</IDL_Type>
+            <Units>unitless</Units>
+            <Count>1</Count>
+        </item>
+        <item>
+            <EFDB_Name>raft_TempControl_gain</EFDB_Name>
+            <Description>loop gain</Description>
+            <IDL_Type>double</IDL_Type>
+            <Units>unitless</Units>
+            <Count>1</Count>
+        </item>
+        <item>
+            <EFDB_Name>raft_TempControl_maxInput</EFDB_Name>
+            <Description>maximum input</Description>
+            <IDL_Type>double</IDL_Type>
+            <Units>unitless</Units>
+            <Count>1</Count>
+        </item>
+        <item>
+            <EFDB_Name>raft_TempControl_maxOutput</EFDB_Name>
+            <Description>maximum PID output (watts)</Description>
+            <IDL_Type>double</IDL_Type>
+            <Units>unitless</Units>
+            <Count>1</Count>
+        </item>
+        <item>
+            <EFDB_Name>raft_TempControl_minInput</EFDB_Name>
+            <Description>minimum input</Description>
+            <IDL_Type>double</IDL_Type>
+            <Units>unitless</Units>
+            <Count>1</Count>
+        </item>
+        <item>
+            <EFDB_Name>raft_TempControl_minOutput</EFDB_Name>
+            <Description>minimum PID output (watts)</Description>
+            <IDL_Type>double</IDL_Type>
+            <Units>unitless</Units>
+            <Count>1</Count>
+        </item>
+        <item>
+            <EFDB_Name>raft_TempControl_rebs</EFDB_Name>
+            <Description>The REBs with heaters to use</Description>
+            <IDL_Type>string</IDL_Type>
+            <Units>unitless</Units>
+            <Count>1</Count>
+        </item>
+        <item>
+            <EFDB_Name>raft_TempControl_setTemp</EFDB_Name>
+            <Description>Temperature set point</Description>
+            <IDL_Type>double</IDL_Type>
+            <Units>unitless</Units>
+            <Count>1</Count>
+        </item>
+        <item>
+            <EFDB_Name>raft_TempControl_smoothTime</EFDB_Name>
+            <Description>input smoothing time (secs)</Description>
+            <IDL_Type>double</IDL_Type>
+            <Units>unitless</Units>
+            <Count>1</Count>
+        </item>
+        <item>
+            <EFDB_Name>raft_TempControl_tempChans</EFDB_Name>
+            <Description>Temperature channels to use</Description>
+            <IDL_Type>string</IDL_Type>
+            <Units>unitless</Units>
+            <Count>1</Count>
+        </item>
+        <item>
+            <EFDB_Name>raft_TempControl_timeConst</EFDB_Name>
```

Comment:
Says (secs) in comment so should that be seconds for units?

Reply:
OK, thanks, these are now generated automatically from CCS, so the real problem is that the units are missing on the original CCS source. I fixed some already but I missed these ones. Will fix it in the XML.

Reply:
Units Fixed.